plugins/services/src/js/containers/service-connection/SDKServiceConnectionContainer.js

```diff
@@ -0,0 +1,270 @@
+          </ConfigurationMapValue>
+        </ConfigurationMapRow>
+        <ConfigurationMapRow>
+          <ConfigurationMapLabel>
+            DNS
+          </ConfigurationMapLabel>
+          <ConfigurationMapValue>
+            {this.getClipboardTrigger(endpoint.getDns())}
+          </ConfigurationMapValue>
+        </ConfigurationMapRow>
+        <ConfigurationMapRow>
+          <ConfigurationMapLabel>
+            VIP
+          </ConfigurationMapLabel>
+          <ConfigurationMapValue>
+            {this.getClipboardTrigger(endpoint.getVip())}
+          </ConfigurationMapValue>
+        </ConfigurationMapRow>
+      </ConfigurationMapSection>
+    );
+  }
+
+  getFileEndpoint(endpoint) {
+    return (
+      <ConfigurationMapRow>
+        <ConfigurationMapLabel>
+          {endpoint.getEndpointName()}
+        </ConfigurationMapLabel>
+        <ConfigurationMapValue>
+          <a
+            className="active endpoint-download"
+            download={endpoint.getEndpointName()}
+            href={`data:text/plain;content-disposition=attachment;filename=${endpoint.getEndpointName()};charset=utf-8,${encodeURIComponent(endpoint.getEndpointData())}`}
+          >
+            <span>
+              <Icon
+                id="download"
+                className="endpoint-download-icon"
+                size="mini"
+              />
+              {" "}
+              Download
+            </span>
+          </a>
+        </ConfigurationMapValue>
+      </ConfigurationMapRow>
+    );
+  }
+
+  getJSONEndpoints(endpoints) {
+    const jsonEndpoints = endpoints.filter(endpoint => {
+      return endpoint.isJSON();
+    });
+
+    if (jsonEndpoints.length === 0) {
+      return null;
+    }
+
+    return jsonEndpoints.map(endpoint => {
+      return this.getJSONEndpoint(endpoint);
+    });
+  }
+
+  getFileEndpoints(endpoints) {
+    const fileEndpoints = endpoints.filter(endpoint => {
+      return !endpoint.isJSON();
+    });
+
+    if (fileEndpoints.length === 0) {
+      return null;
+    }
+
+    return (
+      <ConfigurationMapSection>
+        <ConfigurationMapHeading>
+          Files
+        </ConfigurationMapHeading>
+        {fileEndpoints.map(endpoint => {
+          return this.getFileEndpoint(endpoint);
+        })}
+      </ConfigurationMapSection>
+    );
+  }
+
+  getAlertPanelFooter() {
+    return (
+      <div className="button-collection flush-bottom">
+        <button
+          className="button"
+          onClick={this.handleOpenEditConfigurationModal}
+        >
+          Edit Configuration
+        </button>
+      </div>
+    );
+  }
+
+  getAlertPanelNoEndpoints() {
+    return (
+      <AlertPanel>
+        <AlertPanelHeader>No Endpoints</AlertPanelHeader>
+        <p className="tall">
+          There are no endpoints currently configured for
+          {" "}
+          {this.props.service.getId()}
+          .You can edit the configuration to add service endpoints.
```

Comment:
Trailing space is missing after the first dot

Reply:
Added